_includes/code/schema.things.create.replication.mdx

```diff
@@ -38,28 +38,21 @@ const client = weaviate.client({
   host: 'localhost:8080',
 });
 
-let classObj = {
-    'class': 'Article',
-    'properties': [
+const classObj = {
```

Comment:
I propose to leave it as `let classObj` - all other examples in the docs use `let`, and for me let reflects better the mutability of class schemas.

Reply:
Interesting point about mutability. `let` will cause IDEs to issue a warning to use `const` instead, but I've reverted the change.